frontend/src/metabase/auth/components/AuthLayout.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { Flex } from "grid-styled";
+import Card from "metabase/components/Card";
+
+import AuthScene from "../components/AuthScene";
+import LogoIcon from "metabase/components/LogoIcon";
+
+const AuthLayout = ({ children }) => (
+  <Flex
+    flexDirection="column"
+    flex={1}
+    justifyContent="center"
+    alignItems="center"
+    className="overflow-hidden relative"
+  >
+    <Flex mt={-4} flexDirection="column">
```

Comment:
Why use the grid-styled components over the flex css classes we have?

Reply:
The main reason aside from it just being a bit easier to read / identify certain layout contexts in the component itself (at least I think) is that it allows for some cool things like the negative margin top you see here which helps optically center the logo and card a bit more on the page. We could add our own new `mt-n4` class or something, or just write CSS for that specific case, but that starts to not get very scalable and this has that support baked in already.

In general the "util based" approach to CSS I wrote ages ago (or 5.5 years in FE land at least) still works fine, but it has some downsides like the above. So while I wouldn't say we need to wholesale abandon it, I do think that if we're mucking around and changing layouts significantly, like I did here, it's worthwhile to use some of the styling tools that were designed to work better in a React context.

Reply:
Thanks that's helpful context.